p4rt_app/utils/table_utility.cc

```diff
@@ -15,43 +15,86 @@
 
 #include "absl/status/status.h"
 #include "gutil/status.h"
+#include "absl/strings/substitute.h"
 #include "p4_pdpi/utils/annotation_parser.h"
 #include "swss/schema.h"
 
 namespace p4rt_app {
 namespace table {
 
+// This is what orch agent understands as fixed tables
+std::vector<std::string> FixedTables {
```

Comment:
Lets get rid of this global var, if at all we need it, lets make it a static list of all the fixed tables we have now.

Reply:
I can make it to static. When-ever in future new fixed table is added, we will need to remember to add that entry here. Hope that is okay.